lgsf/commands/base.py

```diff
@@ -178,6 +197,55 @@ def councils_to_run(self):
                 councils.append(council)
         return councils
 
+    def run_councils(self):
+        for council in self.councils_to_run:
+            self.run_council(council)
+
+    def run_councils_with_progress(self):
+        to_run = self.councils_to_run
+        with Progress(
```

Comment:
This has dropped `auto_refresh=False` and a call to `progress.refresh()`. I added them previously as it prevents `ipdb` from working - and it's possible that we'll want to drop in to a shell when working on a scraper. 

Have you tested this case?